src/components/VSelect/mixins/select-watchers.js

```diff
@@ -12,6 +12,11 @@ export default {
       this.$refs.menu && this.$refs.menu.updateDimensions()
     },
     inputValue (val) {
+      // Search for an existing item when a value was selected from the menu
```

Comment:
split comment into two lines. bonus points for having second line 2 less characters than the first

Reply:
Fixed in [f94675dc](https://github.com/rokkie/vuetify/commit/f94675dcaa7163596015a1595831e00255094cf7)